### DISCUSSION TITLE
第2节课作业(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#115] [作业]
**基础作业**：

- 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
- 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。

**进阶作业（可选做）**

- 完成浦语·灵笔的图文理解及创作部署（需截图）
- 完成 Lagent 工具调用 Demo 创作部署（需截图）

**Comment by zhangyutuan** [comment]
https://www.processon.com/view/link/65993b99c61e4f4b1b0db67a

**Comment by jiangye520** [comment]
https://www.processon.com/mindmap/659a9bd93f9b8b77c18be7bb

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec-2

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135460048?spm=1001.2014.3001.5501

**Comment by cemcke** [comment]
http://t.csdnimg.cn/MDRiS

**Comment by admlrz** [comment]
https://blog.csdn.net/weixin_44581312/article/details/135466558

**Comment by Tian-CASIA** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135480850

**Comment by JokerOfRevenge** [comment]
https://www.processon.com/v/659d02c865d15f64a989add7

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135489825?spm=1001.2014.3001.5502

**Comment by adela778** [comment]
https://blog.csdn.net/Adela0546/article/details/135513816

**Comment by MIracleyin** [comment]
https://nnkwjqimr7.feishu.cn/docx/BlnOd3u4soGLcgxVZkZcnHAtnnf?from=from_copylink

**Comment by SChaoWei** [comment]
https://blog.csdn.net/qq_45919032/article/details/135704589?spm=1001.2014.3001.5501

**Comment by hepingpeace** [comment]
https://blog.csdn.net/weixin_55982578/article/details/135985117

**Comment by hepingpeace** [comment]
https://blog.csdn.net/weixin_55982578/article/details/135985117
